i18n/fr/docusaurus-plugin-content-docs/current/misc/On-dust-outputs-and-state-explosion.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position : 20
+title : Sur les poussières d'output et l'explosion des états
```

Comment:
here too

Reply:
fixed in https://github.com/alephium/wiki/pull/39/commits/aa98c0da5a79c88cbe3ac669aeaa6acb4a6c3f8f